src/SpotifyWebAPI.php

```diff
@@ -364,15 +365,15 @@ public function changeVolume($options)
      *
      * @return array|object The new playlist. Type is controlled by the `return_assoc` option.
      */
-    public function createPlaylist($options)
+    public function createPlaylist($userId, $options)
```

Comment:
I'm afraid this would be a breaking change.

We could add a check to keep the backwards compatibility for a while and remove that check in the future, for example: 

```suggestion
    public function createPlaylist($userId, $options = [])
{
    if (is_array($userId) || is_object($userId)) {
        $options = $userId;
        $userId = 'me';
    }
```

Reply:
i've added your suggestions